NextcloudTalk/Chat/Chat cells/AutoCompletionTableViewCell.swift

```diff
@@ -0,0 +1,120 @@
+        let button = AvatarButton(frame: .init(x: 0, y: 0, width: avatarHeight, height: avatarHeight))
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = NCAppBranding.placeholderColor()
+        button.layer.cornerRadius = avatarHeight / 2
+        button.layer.masksToBounds = true
+        button.showsMenuAsPrimaryAction = true
+        button.imageView?.contentMode = .scaleToFill
+
+        return button
+    }()
+
+    private lazy var userStatusImageView = {
+        let imageView = UIImageView(frame: .init(x: 0, y: 0, width: 12, height: 12))
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.isUserInteractionEnabled = false
+
+        return imageView
+    }()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        self.backgroundColor = .secondarySystemBackground
+        self.configureSubviews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+
+        self.titleLabel.font = .preferredFont(forTextStyle: .body)
+        self.titleLabel.text = ""
+
+        self.avatarButton.cancelCurrentRequest()
+        self.avatarButton.setImage(nil, for: .normal)
+
+        self.userStatusImageView.image = nil
+        self.userStatusImageView.backgroundColor = .clear
+    }
+
+    func configureSubviews() {
+        self.contentView.addSubview(self.avatarButton)
+        self.contentView.addSubview(self.userStatusImageView)
+        self.contentView.addSubview(self.titleLabel)
+
+        let views = [
+            "avatarButton": self.avatarButton,
+            "userStatusImageView": self.userStatusImageView,
+            "titleLabel": self.titleLabel
+        ]
+
+        let metrics = [
+            "avatarSize": avatarHeight,
+            "right": 10,
+        ]
+
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-right-[avatarButton(avatarSize)]-right-[titleLabel]-right-|", metrics: metrics, views: views))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[titleLabel]|", metrics: metrics, views: views))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-32-[userStatusImageView(12)]-(>=0)-|", metrics: metrics, views: views))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-32-[userStatusImageView(12)]-(>=0)-|", metrics: metrics, views: views))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-right-[avatarButton(avatarSize)]-(>=0)-|", metrics: metrics, views: views))
+    }
+
+    public func setUserStatus(_ status: String) {
+        var statusImage: UIImage? = nil
```

Comment:
```suggestion
        var statusImage: UIImage?
```
Fix SwiftLint warning